java/src/main/java/org/rocksdb/TickerType.java

```diff
@@ -876,6 +876,8 @@ public enum TickerType {
 
     PREFETCH_HITS((byte) -0x53),
 
+    SST_FOOTER_CORRUPTION_COUNT((byte) -0x55),
+
     TICKER_ENUM_MAX((byte) -0x54);
```

Comment:
Should this value be updated after the addition of `SST_FOOTER_CORRUPTION_COUNT`?